recording/src/nextcloud/talk/recording/Config.py

```diff
@@ -236,4 +237,11 @@ def getFfmpegExtensionVideo(self):
         """
         return self._configParser.get('ffmpeg', 'extensionvideo', fallback='.webm')
 
+    def getBrowserForRecording(self):
+        """
+        Returns the browser identifier that will be used for recordings.
+        Defaults to "firefox".
+        """
+        return self._configParser.get('recording', 'browser', fallback='firefox')
+
```

Comment:
The changes in recording/src/nextcloud/talk/recording/Config.py should be moved from the first commit to the second commit, which is the one that adds the setting.